docs/nx-cloud/private/auth-gitlab.md

```diff
@@ -24,22 +29,22 @@ Ensure you grant it the "`read_user`" scope:
 
 ![Step 4](/nx-cloud/private/images/gitlab_step_4.png)
 
-Click "Save".
+Click "Save application".
 
 Once you create, keep a note of the Client ID and the Secret:
 
 ![Step 5](/nx-cloud/private/images/gitlab_step_5.png)
 
 ## Connect your Nx Cloud installation to your new app
 
-Provide the following env variables to the `nx-cloud-api` container:
+Provide the following env variables to the `nx-cloud-frontend` container:
 
 - `GITLAB_APP_ID`
 - `GITLAB_APP_SECRET`
```

Comment:
I should've done this when I initially added the Helm docs (I guess I assumed people would start from Helm): https://github.com/nrwl/nx-cloud-helm/blob/main/AUTH-GUIDE.md

but should we add a reference to those Helm config options? basically if they're using Helm they shouldn't have to worry about where to add these env values

Reply:
Something like "If you're using our Helm chart, you'll find all the information you need regarding required env variables in [that repository](https://github.com/nrwl/nx-cloud-helm/blob/main/AUTH-GUIDE.md)."?